include/BLE_Common.h

```diff
@@ -49,8 +49,8 @@
 
 // macros to convert different types of bytes into int The naming here sucks and
 // should be fixed.
-#define bytes_to_s16(MSB, LSB) (((signed int)((signed char)MSB))) << 8 | (((signed char)LSB))
-#define bytes_to_u16(MSB, LSB) (((signed int)((signed char)MSB))) << 8 | (((unsigned char)LSB))
+#define bytes_to_s16(MSB, LSB) (((int16_t)((signed char)MSB))) << 8 | (((signed char)LSB))
+#define bytes_to_u16(MSB, LSB) (((uint16_t)((signed char)MSB))) << 8 | (((unsigned char)LSB))
```

Comment:
Good catch - This looks correct but since it's a significant change (signed int to uint16), is anyone able to confirm we don't accidently do weird things around the 128's . Specifically greater than 7.6% grade or >768 watts.

Reply:
Maybe just change all `bytes_to_u16` to use `bytes_to_int`? That effectively leaves them unchanged, less testing worry that way?

Reply:
I like that from a testing standpoint. Otherwise we should write some tests to cover the change.

Reply:
> Maybe just change all `bytes_to_u16` to use `bytes_to_int` (leaving `CyclePowerData.cpp` using `bytes_to_u16` obviously)? That effectively leaves them unchanged, less testing worry that way?

I like your idea of changing it to uint and int where required. It makes the rest of the code and what's happening more clear.

Reply:
I had to revert this because it was throwing impossible numbers in my testing.